homeassistant/components/config/device_registry.py

```diff
@@ -35,5 +47,28 @@
             'sw_version': entry.sw_version,
             'id': entry.id,
             'hub_device_id': entry.hub_device_id,
+            'area_id': entry.area_id,
         } for entry in registry.devices.values()]
     ))
+
+
+@async_response
+async def websocket_update_device(hass, connection, msg):
+    """Handle update area websocket command."""
+    registry = await async_get_registry(hass)
+
+    entry = registry.async_update_device(
+        msg['device_id'], area_id=msg['area_id'])
+
+    connection.send_message(websocket_api.result_message(
+        msg['id'], _entry_dict(entry)
+    ))
+
+
+@callback
+def _entry_dict(entry):
```

Comment:
This is wrong, we should return the same rendering as we do for list.

Reply:
Allright, I assumed all that was need was the diff. Will fix!